earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Sorry, I meant the webpage 'Install-Fusion-or-Earth-Server' - does that have a different review cycle?